### DISCUSSION TITLE
General Troubleshooting Guide?

### DISCUSSION THREAD

**Discussion by alfista2600** [sle118/squeezelite-esp32#380] [Q&A]
Is there a general list of troubleshooting steps somewhere? I have a cheap board from amazon that works for about 30 minutes and then stops... it might reconnect/might not. I'm streaming from Apple's Music app on Sonoma which always seems to get confused when moving between networks so I'm not sure where to look for issues first. I want to replace a handful of airplay devices with these once I can get it going. I do not use LMS.

**Comment by sle118** [comment]
https://github.com/sle118/squeezelite-esp32/issues/new/choose

Look at the issues template. Documentation best practices are all there.
